app/cafe.py

```diff
@@ -0,0 +1,33 @@
+import datetime

```

Comment:
from datetime you need only `date`, not all module

Reply:
![py_lockdown_cafe](https://github.com/user-attachments/assets/03511e4d-4e36-4f5f-8a73-6610807412fc)

it's not working if I use only date

Reply:
if expiration_date < date.today()